apps/rush/src/actions/CheckVersionsAction.ts

```diff
@@ -37,6 +38,11 @@ export default class CheckVersionsAction extends CommandLineAction {
 
     const config: RushConfiguration = RushConfiguration.loadFromDefaultLocation();
 
+    if (config.approvedPackagesPolicyEnabled) {
+      const packageReviewChecker: PackageReviewChecker = new PackageReviewChecker(config);
+      packageReviewChecker.saveCurrentDependencies();
+    }
```

Comment:
Actually, I just remembered that "rush install" is now a no-op, so we can use that.  I'll remove it from here.

---
In reply to: [116136740](https://github.com/Microsoft/web-build-tools/pull/181#discussion_r116136740) [](ancestors = 116136740,116122167,116119876)